### PR TITLE
[MI-2435] Updated documentation to add the steps for creating a exchange service account.

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,27 @@ The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service t
 
 ## Setup
 
+### Create an Exchange Service Account([Reference](https://www.cirrusinsight.com/knowledgebase/exchange-impersonated-sa))
+
+1. Go to you exchange admin center.(https://your-exchange-server-url/ecp)
```
Comment (Kshitij-Katiyar) [inline README.md:22]:
```suggestion
1. Go to your exchange admin center.(https://your-exchange-server-url/ecp)
```

File: README.md [hunk 2]
```diff
@@ -17,15 +17,27 @@ The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service t
...
+2. Click on 'recipients' in the navigation panel.
+3. Click the + and select the 'User mailbox' option to create the new service account.
```
Comment (Kshitij-Katiyar) [inline README.md:24]: If this is an icon
```suggestion
3. Click the + icon and select the 'User mailbox' option to create the new service account.
```

File: README.md [hunk 3]
```diff
@@ -17,15 +17,27 @@ The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service t
...
+4. Select the 'New user' option and complete the form.
+5. Click on 'permissions' in the navigation panel.
+6. Click the + to add a new Role Group. Enter a value for Name and Description. Leave the 'Write scope' value set to 'Default’.
```
Comment (Kshitij-Katiyar) [inline README.md:27]: Maybe ?
```suggestion
6. Click on the + icon to add a new Role Group. Enter a value for Name and Description. Leave the 'Write scope' value set to 'Default’
```

File: README.md [hunk 4]
```diff
@@ -17,15 +17,27 @@ The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service t
...
+
+1. Go to your exchange admin center.(https://your-exchange-server-url/ecp)
+2. Click on 'recipients' in the navigation panel.
+3. Click on the + icon and select the 'User mailbox' option to create the new service account.
+4. Select the 'New user' option and complete the form.
+5. Click on 'permissions' in the navigation panel.
+6. Click on the + icon to add a new Role Group. Enter the values for Name and Description. Leave the 'Write scope' value set to 'Default’
+7. Click on the + icon under 'Roles' and add 'ApplicationImpersonation’. Click 'OK' once it has been added to the list.
+8. Click on the + icon under 'Members' and add the service account you created. Click 'OK' once it has been added to the list.
+9. After completing the form, click the 'Save' button and the new role group should be added to your list.
```
Comment (Kshitij-Katiyar) [inline README.md:30]:
```suggestion
9. After completing the form, click on the 'Save' button and the new role group should be added to your list.
```

**Review by Kshitij-Katiyar — APPROVED** [review]

**Review by ayusht2810 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -17,15 +17,27 @@ The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service t
 
 ## Setup
 
+### Create an Exchange Service Account([Reference](https://www.cirrusinsight.com/knowledgebase/exchange-impersonated-sa))
+
+1. Go to your exchange admin center.(https://your-exchange-server-url/ecp)
```
Comment (manojmalik20) [inline README.md:22]:
```suggestion
1. Go to your Exchange admin center.(https://your-exchange-server-url/ecp)
```

**Review by manojmalik20 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**